src/_pytest/python.py

```diff
@@ -125,9 +133,10 @@ def pytest_cmdline_main(config):
         return 0
 
 
-def pytest_generate_tests(metafunc):
+def pytest_generate_tests(metafunc: "Metafunc") -> None:
     for marker in metafunc.definition.iter_markers(name="parametrize"):
-        metafunc.parametrize(*marker.args, **marker.kwargs, _param_mark=marker)
+        # TODO: Fix this type-ignore (overlapping kwargs).
```

Comment:
Do you have a plan for this?

Reply:
Not really. There's a workaround of creating `kwargs = {**marker.kwargs, '_param_marker': marker}` but I think it's worse than the disease.